src/music.ts

```diff
@@ -206,7 +207,7 @@ export class Music {
     }
   }
 
-  getKeyParts(keyString) {
+  getKeyParts(keyString: string): MusicNoteParts {
```

Comment:
What is the reason why the in this function the accidentals n,bb,## are not allowed? 
`const regex = /^([cdefgab])(b|#)?(mel|harm|m|M)?$/;`

Reply:
I suggest to remove this function and include the `(mel|harm|m|M)` in `getNoteParts` returning n for accidental and M for type, when they are not provided.